app/views/administration/index/_my_lectures.html.erb

```diff
@@ -42,7 +42,7 @@
     <% if current_user.lectures_as_module_editor.present? %>
       <%= t('admin.main.as_course_editor') %>
       (<%= current_user.lectures_as_module_editor.count %>)
-      <%= helpdesk(t('admin.main.info.lectures_as_course_editor'), false) %>
+      <%= helpdesk(t('admin.main.info.lectures_as_module_editor'), false) %>
```

Comment:
I would prefer to leave the naming as it is, since it refers to the `course` model. As a consequence there is probably some line in the `.yml` file that needs changing.

Reply:
There is also `lectures_as_module_editor` in `user.rb`. should this be also renamed?

Reply:
Yes, that makes sense.